lib/pascal_triangle.rb

```diff
@@ -0,0 +1,27 @@
+class PascalTriangle
+
+  def print_lines_to(line_number)
+    all_lines = []
+    lines_numbers = (0..line_number).map {|number| number}
```

Comment:
Is there any particular reason we need to `map` from number --> number? I wonder if you could do lines 5 and 6 in one go?